analytics/src/analytics/integrations/db.py

```diff
@@ -22,8 +21,10 @@ def get_db() -> Engine:
     sqlalchemy.engine.Engine
     A SQLAlchemy engine object representing the connection to the database.
     """
+    db = DBSettings
```

Comment:
You'd want to use `get_db_settings` here as well.

Reply:
@chouinar do you mean something like 
```python
def get_db_settings() -> DBSettings:
     return DBSettings()

def get_db() -> Engine:
# rest of code

```

Reply:
In a few other files you do:

```py
from config import get_db_settings

settings = get_db_settings()
```

You need to do that here as well.

Just doing `DBSettings.user` isn't valid code - a quick test shows that would raise an AttributeError. DBSettings is the class, not an actual settings object.

--

Is this bit of code actually running in the tests? It should fail if run as written right now

Reply:
@aplybeah any update on the suggested change?